package/yast2-packager.spec

```diff
@@ -25,17 +25,20 @@ Source0:        %{name}-%{version}.tar.bz2
 
 Url:            https://github.com/kobliha/yast-packager
 BuildRequires:  update-desktop-files
-BuildRequires:  yast2-country-data
 BuildRequires:  yast2-devtools >= 3.1.10
-BuildRequires:  yast2-storage
-BuildRequires:  yast2-testsuite
 BuildRequires:  yast2-xml
-BuildRequires:  yast2_theme
+# needed for space calculator to parse and convert sizes
```

Comment:
NP: I'd add a `TODO` note here, depending on a full storage stack just because of parsing size strings does not look nice to me. Getting rid of this dependency would help quite a lot I guess...